lib/convection/model/template/resource_property/aws_s3_replication_configuration.rb

```diff
@@ -0,0 +1,22 @@
+require_relative '../resource_property'
+
+module Convection
+  module Model
+    class Template
+      class ResourceProperty
+        # Represents an {http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket-replicationconfiguration.html
+        # Amazon S3 Replication Configuration}
+        class S3ReplicationConfiguration < ResourceProperty
+          property :role, 'Role'
+          property :rules, 'Rules'
```

Comment:
This should be declared as a list or the `rules` method will not respond to `<<`.